Library/Formula/mariadb.rb

```diff
@@ -80,6 +80,11 @@ def install
       args << "-DPLUGIN_TOKUDB=NO"
     end
 
+    # disable Mroonga on 10.1
```

Comment:
> But I think I found a fix.

Great! If it's something that can be reported upstream, it'd be great if you could do that too.